### ISSUE TITLE
NameError on installation

### ISSUE THREAD

**Issue by sils** [fedora-infra/asknot-ng#28]
```
Traceback (most recent call last):
  File "./asknot-ng.py", line 161, in <module>
    main(**args)
  File "./asknot-ng.py", line 128, in main
    work(_=_, lang=lang, languages=languages, **kw)
  File "./asknot-ng.py", line 44, in work
    data = load_yaml(question_filename)
  File "/home/lasse/prog/asknot-ng/asknot_lib.py", line 47, in load_yaml
    validate_yaml(data, basedir)
  File "/home/lasse/prog/asknot-ng/asknot_lib.py", line 59, in validate_yaml
    validate_tree(data['tree'], basedir)
  File "/home/lasse/prog/asknot-ng/asknot_lib.py", line 71, in validate_tree
    if isinstance(node['children'], basestring):
NameError: name 'basestring' is not defined
```

looks reasonable from code...

**Comment by ralphbean** [comment]
Oh, you must be using python3!  If you use python2, it should work (on the other hand, we should support python3 -- this is just a bug).

**Comment by sils** [comment]
plus I did the virtualenv stuff so you should probably explicitly use
python2

2015-08-15 16:05 GMT+02:00 Ralph Bean notifications@github.com:

**Comment by ralphbean** [comment]
OK - this is fixed in 6b2b1dc605 and #34 finishes off python3 support.

**Comment by sils** [comment]
I can confirm the fix btw. :)
